base/atomic_sequence_num.h

```diff
@@ -20,7 +20,11 @@ class AtomicSequenceNumber {
 
   // Returns an increasing sequence number starts from 0 for each call.
   // This function can be called from any thread without data race.
-  inline int GetNext() { return seq_.fetch_add(1, std::memory_order_relaxed); }
+  inline int GetNext() {
+    int result = seq_.fetch_add(1, std::memory_order_relaxed);
+    result = recordreplay::RecordReplayValue("base::AtomicSequenceNumber", result);
+    return result;
+  }
```

Comment:
This change seems risky.  Have you verified that every single use of this class is in code which we expect to run deterministically?  If not this RecordReplayValue should be in the caller.

Reply:
That's a good point. I forget that `RecordReplayValue` will fail if events are not allowed.
Will check!